simulationworkflowschema/equation_of_state.py

```diff
@@ -161,12 +160,17 @@ def normalize(self, archive, logger):
             self.results = EquationOfStateResults()
             self.outputs.append(Link(name=WORKFLOW_RESULTS_NAME, section=self.results))
 
-        task0_archive = self.tasks[0].task.m_root()
-        tasks = self.tasks[:]
-        if task0_archive.workflow2:
-            # remove the first task if it is a GeometryOptimization (not part of the EOS)
-            if isinstance(task0_archive.workflow2, GeometryOptimization):
-                tasks.pop(0)
+        try:
+            task_archives = [task.task.m_root() for task in self.tasks]
```

Comment:
i do not understand this, these are the archives and not the workflow2 (task) section.